utils/create-jobs.py

```diff
@@ -16,7 +16,7 @@ def parse_cmdline(machines, tests, rfs_types):
     parser.add_argument('--urlbase', '-u', action='store', dest='urlbase',
                         help='url fetch base',
                         default='https://download.automotivelinux.org/AGL/upload/ci/')
-    parser.add_argument('--boot', action='store', dest='rfs_type', nargs = 1,
+    parser.add_argument('--boot', action='store', dest='rfs_type',
```

Comment:
nargs doesn't "requires" arguments, it just limits the number of arguments you can pass to --boot.
To require arguments argparse uses `"required=True"`
Boot was not required and with this change you're allowing to pass multiple boot args in the command line. Not sure it's what you wanted to do.

Reply:
Are you sure that I'm allwing multiple argument. I think the default is one ... which is why, w/o nargs, rfs_type is no longer a list

Reply:
Oh my bad, did not know the default was one. Indeed not useful then :)
It's just your commit message then, it did not change the requirement of boot parameter.

Reply:
Hum, I indented it to be required which is why I added nargs to begin with but, I messed and it was indeed not required by argparse.
That being said, the generated job would be rubbish withtout it ... so in way, it was required but not enforced :stuck_out_tongue_winking_eye: